src/components/Modal.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components'
+
+const Modal = ({ state, statechange }) => {
+  return (
+    <>
+      {state && (
+        <Overlay>
+          <Container>
+            <ModalInfo>Insert Modal Message Here</ModalInfo>
+            <AcceptButton onClick={() => statechange(false)}>Insert Button Text Here</AcceptButton>
+          </Container>
+        </Overlay>
+      )}
+    </>
```

Comment:
I think maybe it would be good to use div instead of </> since I see that the other components are like this.

Reply:
No problem for me!